.github/workflows/ci-test-coverage.yml

```diff
@@ -0,0 +1,33 @@
+name: ci-test-coverage
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "KubeArmor/**"
+      - "tests/**"
+      - "protobuf/**"
+      - ".github/workflows/ci-test-coverage.yml"
+  pull_request:
+    branches: [main]
+    paths:
+      - "KubeArmor/**"
+      - "tests/**"
+      - "protobuf/**"
+      - ".github/workflows/ci-test-coverage.yml"
+
+jobs:
+    codecoverage:
```

Comment:
Also we somehow need to run this only after other two complete. Currently it's faling because this workflow starts running immediately and doesn't wait for the other 2.

Ig can replace the events to `workflow_run`
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run

You will need to dig how exactly to work this out.